src/api/entities/Instruction/index.ts

```diff
@@ -355,7 +355,7 @@ export class Instruction extends Entity<UniqueIdentifiers, string> {
         },
       },
     } = await context.queryMiddleware<Ensured<Query, 'instructions'>>(
-      instructionsQuery(false, {
+      instructionsQuery(context.isSqIdPadded, {
         id: id.toString(),
```

Comment:
In SQ 19 instruction IDs also are padded so this needs conditional padding.

Reply:
Thanks. I added handling code in the query now